pos_multiple_control_automatic_solve/models/account_bank_statement.py

```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import Warning as UserError
+
+
+class AccountBankStatement(models.Model):
+    _inherit = 'account.bank.statement'
+
+    pos_session_state = fields.Char(
+        string='Pos session state',
+        compute='_compute_pos_session_state')
+
+    display_autosolve = fields.Boolean(
+        string='Control Difference',
+        compute='_compute_display_autosolve')
+
+    # Compute Section
+    @api.one
+    def _compute_pos_session_state(self):
+        self.pos_session_state = self.pos_session_id.state
+
+    @api.one
```

Comment:
``@api.one`` est déprécié dans les versions futurs.

plutot mettre 

    @api.multi
    def my_function(self):
        self.ensure_one()
        # use self to have your object.